apps/blog/models/blog.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from apps.blog.models.author import Author
+from apps.blog.models.tag import Tag
+from apps.blog.models.category import Category
+
+class Blog(models.Model):
```

Comment:
What we need now here is a uuid for the primary key
```python
from apps.users.models.base_model import BaseModel


class Blog(BaseModel):
```

After that please proceed to `makemigrations` command.